core/redis_client.py

```
@@ -0,0 +1,443 @@
+        except Exception as e:
+            logger.error(f"Failed to queue onboarding for {mobile_number}: {e}", exc_info=True)
+            return False
+
+    async def add_to_queue_input_sms(
+        self,
+        redis_id: int,
+        mobile_number: str,
+        device_id: str,
+        sms_message: str,
+        received_timestamp: str,
+        country_code: str = None,
```

Ruff (0.14.8)

PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)